src/EUTelProcessorPatternRecognition.cc

```diff
@@ -157,7 +157,6 @@ void EUTelProcessorPatternRecognition::processRunHeader(LCRunHeader* run) {
 				<< "The run header says the GeoID is " << header->getGeoID() << std::endl
 				<< "The GEAR description says is     " << geo::gGeometry().getSiPlanesLayoutID() << std::endl;
 	}
-	_trackFitter->clearEveryRun();//This just sets the counter for the total number of hits/total number of shared hits to zero. 		
```

Comment:
@AlexanderMorton As discussed above, this does exactly nothing! Fix it if needed!